test/test_sparse.py

```diff
@@ -362,28 +362,27 @@ def test_transpose(self):
             y = y.transpose(i, j)
             self.assertEqual(self.safeToDense(x), y)
 
-    def test_transpose_coalesce_invariant(self):
-        # If a sparse tensor is coalesced, its transpose should be the same
-        # If a sparse tensor is uncoalesed, its transpose should be the same
-        x_coalesced = self._gen_sparse(2, 3, 4)[0].coalesce()
-        x_indices = x_coalesced._indices()
-        x_values = x_coalesced._values()
-
-        y_uncoalesced = self.SparseTensor(
-            torch.cat([x_indices, x_indices], dim=1),
-            torch.cat([x_values, x_values]),
-            x_coalesced.size())
-
-        self.assertTrue(x_coalesced.is_coalesced())
-        self.assertFalse(y_uncoalesced.is_coalesced())
-
-        self.assertTrue(x_coalesced.transpose(0, 1).is_coalesced())
-        self.assertFalse(y_uncoalesced.transpose(0, 1).is_coalesced())
-
-        x_coalesced.transpose_(0, 1)
-        y_uncoalesced.transpose_(0, 1)
-        self.assertTrue(x_coalesced.is_coalesced())
-        self.assertFalse(y_uncoalesced.is_coalesced())
+    def test_transpose_coalesce(self):
```

Comment:
Please restore the original test, and create a new method for this one. It's not checking `transpose_coalesce`

Reply:
The original test is restored with a slight modification.